cranelift/codegen/src/isa/x64/pcc.rs

```diff
@@ -163,6 +163,7 @@ pub(crate) fn check(
             op: _,
             ref src1_dst,
             src2: _,
+            ..
```

Comment:
We try to avoid any `..` here so we don't miss new fields being added. Could we instead do `lock: _`?